MOxUnit/@MOxUnitTestResult/MOxUnitTestResult.m

```diff
@@ -34,5 +34,7 @@
     s.skips=cell(0);
     s.successes=cell(0);
     s.testsRun=0;
+    s.duration=0;
+    s.xml_reports=cell(0);
```

Comment:
If XML needs a header to be printed before any test result, then the report function should, when output is set to XML, check to see if any tests has already been run - and if not, print the XML header.